classes/class-kp-session.php

```diff
@@ -63,7 +63,7 @@ public function __construct() {
 	 * @param int|WC_Order|null $order The WooCommerce order or order id. Null if we are working with a cart.
 	 */
 	public function get_session( $order = null ) {
-		if ( ! kp_is_available() || ! kp_is_checkout_page() && ! KP_Subscription::is_change_payment_method() ) {
+		if ( ! kp_is_checkout_page() || ! kp_is_available() && ! KP_Subscription::is_change_payment_method() ) {
```

Comment:
The changes look good! However, can you adjust the `if` to look like this. When I initially wrote the code, I wasn't explicit about this.

```php
		if ( ( ! kp_is_available() || ! kp_is_checkout_page() ) && ! KP_Subscription::is_change_payment_method() ) {
```

Reply:
Absolutely, should now be adjusted.